frontend/src/common/enums/app/app-route.enum.ts

```diff
@@ -8,6 +8,7 @@ enum AppRoute {
   BS = '/bs',
   BS_CREATE_SPACE = '/bs/create-space',
   SC = '/sc',
+  SC_CREATE_INSTANCE = '/sc/create-instance',
```

Comment:
```suggestion
  SC_CREATE_INSTANCE = '/sc/configurate-instance',
```
since in the future we will be mange to change it